cache/cache.go

```diff
@@ -56,6 +56,8 @@ type Config struct {
 	// only set this flag to true when testing or throughput performance isn't a
 	// major factor.
 	Metrics bool
+	// NotEvictNew determines if new k-v can be evict when Set
+	NotEvictNew bool
```

Comment:
`EvictNew` is easier to read.

Reply:
We need to introduce something like `DefaultConfig` to enable `EvictNew`.